client/components/loading-placeholder/index.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import noop from 'lodash/utility/noop';
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
+
+const LoadingPlaceholder = React.createClass( {
+	propTypes: {
+		className: React.PropTypes.string
+	},
+
+	goBack() {
+		page.back( '/' );
```

Comment:
This should probably be passed in the props, to make it more reusable. I assume you have other use cases in mind.